.changeset/pretty-cheetahs-taste.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏this should be a patch

I know the changes should be no-ops, but it's technically possible for them to break something or change behavior.